src/main/java/bio/terra/service/filedata/google/firestore/FireStoreFileDao.java

```diff
@@ -205,6 +219,13 @@ List<FireStoreFile> enumerateAll(Firestore firestore, String datasetId)
     return fireStoreUtils.query(fileColl, FireStoreFile.class);
   }
 
+  List<FireStoreFile> enumerateAllWithEmptyField(
+      Firestore firestore, String datasetId, String fieldName) throws InterruptedException {
+    Query fileColl =
+        firestore.collection(makeCollectionId(datasetId)).whereEqualTo(fieldName, null);
```

Comment:
Does Firestore distinguish between a null field and any other definition of "empty" i.e. an empty string?